services/navigation/builtin/builtin_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"go.viam.com/rdk/services/navigation"
 	"go.viam.com/rdk/services/slam"
 	fakeslam "go.viam.com/rdk/services/slam/fake"
+	_ "go.viam.com/rdk/services/vision"
+	_ "go.viam.com/rdk/services/vision/colordetector"
```

Comment:
I think you should only need the first one?

Reply:
To follow on this, it is Bad Practice to import things unnamed in this way unless there's a good reason to do so.

Reply:
> it is Bad Practice to import things unnamed in this way unless there's a good reason to do so.

I agree with you. Without this import line:
```
_ "go.viam.com/rdk/services/vision/colordetector"
```

the `TestNavSetup` test fails for me with the following log:
```
    logger.go:130: 2023-07-31T10:19:33.433-0400 DEBUG   impl/resource_manager.go:544    now configuring resource        {"resource": "rdk:service:navigation/test_navigation"}
    logger.go:130: 2023-07-31T10:19:33.434-0400 ERROR   impl/resource_manager.go:573    error building resource {"resource": "rdk:service:navigation/test_navigation", "model": "rdk:builtin:builtin", "error": "dependency \"blue_square\" is not ready yet; reason=\"resource \\\"rdk:service:vision/blue_square\\\" not available; reason=\\\"resource build error: unknown resource type: rdk:service:vision and/or model: rdk:builtin:color_detector\\\"\""}
    logger.go:130: 2023-07-31T10:19:33.434-0400 DEBUG   framesystem/framesystem.go:152  updated robot frame system:
        +---+-------+--------+-------------+-------------+----------+
        | # | NAME  | PARENT | TRANSLATION | ORIENTATION | GEOMETRY |
        +---+-------+--------+-------------+-------------+----------+
        | 0 | world |        |             |             |          |
        +---+-------+--------+-------------+-------------+----------+
    builtin_test.go:50: Expected: nil
        Actual:   'resource "rdk:service:navigation/test_navigation" not available; reason="resource build error: dependency \"blue_square\" is not ready yet; reason=\"resource \\\"rdk:service:vision/blue_square\\\" not available; reason=\\\"resource build error: unknown resource type: rdk:service:vision and/or model: rdk:builtin:color_detector\\\"\""'
--- FAIL: TestNavSetup (0.00s)
```